src/http/lib.ts

```diff
@@ -1,26 +1,8 @@
 import type { ParsedQuery } from 'query-string';
 import queryString from 'query-string';
-import * as nodeUrl from 'url';
 
 import type { DeferredJobResponse } from './types';
 
-/**
- * Choose which `URL` class to use based on the environment (Browser / Node.js / Edge)
- */
-function getURLClass() {
```

Comment:
Are you sure this won't break things when using SDK in Edge environment? I think when I was adding this code, we were already using Node 16, so it might be that applications that run on Node 16 might break.

Reply:
> Are you sure this won't break things when using SDK in Edge environment?

Tried running it in the Cloudflare Worker Edge environment. I confirm it works:

![Screenshot from 2023-12-11 16-24-43](https://github.com/prezly/javascript-sdk/assets/370680/0fcc24dd-eeae-49ab-8891-541d3e7c4d40)

Reply:
> so it might be that applications that run on Node 16 might break.

True. But wouldn't it be solvable by polyfilling the `URL` API in the specific environment, similar to `fetch`?